### ISSUE TITLE
mac always a string under python3

### ISSUE THREAD

**Issue by jlec** [tubaman/django-macaddress#19]
Under python3 I see the following although I have set ```MACADDRESS_DEFAULT_DIALECT = 'netaddr.mac_eui48'```

```
In [1]: from infrabrowser.models import Unit
In [2]: Unit.objects.first().mac
Out[2]: '207369326246'
```
Following patch (suggested by 2to3) fixes this issue 
```
--- /usr/local/lib/python2.7/site-packages/macaddress/fields.py	(original)
+++ /usr/local/lib/python2.7/site-packages/macaddress/fields.py	(refactored)
@@ -9,10 +9,9 @@

 import warnings

-class MACAddressField(models.Field):
+class MACAddressField(models.Field, metaclass=models.SubfieldBase):
     description = "A MAC address validated by netaddr.EUI"
     empty_strings_allowed = False
-    __metaclass__ = models.SubfieldBase
     dialect = None

     def __init__(self, *args, **kwargs):
```

**Comment by kra3** [comment]
Hmm, this is new syntax. let me verify whether it will work for python 2.x series as well.
Else have to find a work around.

**Comment by kra3** [comment]
Have to use `with_metaclass` from `six` or `future.utils` to maintain backward compatibility.

Thanks @jlec for pointing this out.

**Comment by kra3** [comment]
Fix has been submitted with pull request #21. Therefor Closing. @jlec please try the latest repo checkout.
